src/override/malloc.cc

```diff
@@ -43,7 +43,8 @@ extern "C"
     return ThreadAlloc::get_noncachable()->alloc<ZeroMem::YesZero>(sz);
   }
 
-  SNMALLOC_EXPORT size_t SNMALLOC_NAME_MANGLE(malloc_usable_size)(void* ptr)
+  SNMALLOC_EXPORT
+    size_t SNMALLOC_NAME_MANGLE(malloc_usable_size)(const void* ptr)
```

Comment:
I think you still need the non-const version for other platforms that declare this in a header that we include here.

Reply:
Currently, it seems to work for other platforms.

I guess the case seems to be this:
- Android somehow includes the `malloc.h` via the headers.
- Other platforms does not do this.
- Compiler discards CV-qualifiers so linking stage is not affected for most platforms.
- However, the definitions conflicts on android because the compiler can find the real definitions.

So, do you still suggest me to provide two different versions? if so, I will work on it ASAP.

Reply:
@davidchisnall How about something like:

```cmake
include(CheckCSourceCompiles)
CHECK_C_SOURCE_COMPILES("
#include <malloc.h>
size_t malloc_usable_size(const void* ptr) { return 0; }
int main() { return 0; }
" CONST_MALLOC_USABLE_SIZE)
```